sycl/source/detail/scheduler/commands.cpp

```diff
@@ -197,7 +201,15 @@ class DispatchHostTask {
     CGHostTask &HostTask = static_cast<CGHostTask &>(MThisCmd->getCG());
 
     // we're ready to call the user-defined lambda now
-    HostTask.MHostTask->call();
+    if (HostTask.MHostTask->isInteropTask()) {
+      interop_handle IH{MReqToMem, HostTask.MQueue,
+                        getSyclObjImpl(HostTask.MQueue->get_device()),
+                        HostTask.MQueue->getContextImplPtr()};
+
+      HostTask.MHostTask->call(IH);
+    } else
```

Comment:
Minor comment: I may be wrong, but the LLVM coding standard asks to be consistent with braces. At least I had such comment to my code previously:
Use 'if (A) {X1; X2; } else {Y;}' instead of 'if (A) {X1; X2; } else Y;'

Reply:
According to [this paragraph](https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements) it's OK to have a one-line statement not enclosed in braces if the statement is pretty simple.
Are you OK with that?

Reply:
Hmm, Ok.